schema/site_metadata.json

```diff
@@ -0,0 +1,74 @@
+{
+    "$udmi_version": "1.5.0",
+    "title": "Site Metadata",
+    "$section": "Site Model",
+    "type": "object",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "additionalProperties": false,
+    "required": [
+      "timestamp",
+      "version"
+    ],
+    "properties": {
+      "timestamp": {
+        "description": "RFC 3339 timestamp the site metadata was last updated",
+        "type": "string",
+        "format": "date-time",
+        "examples": ["2019-01-17T14:02:29.364Z"]
+      },
+      "version": {
+        "description": "Version of the UDMI schema",
+        "type": "string"
+      },
+      "id": {
```

Comment:
please something more specific than "id" -- like "test" or anything very basic it ends up being horribly confusing. Maybe site_id?  or site_code?